docs/FAQ_RU.md

```diff
@@ -273,3 +273,112 @@ spec:
     - kind: VirtualDisk
       name: win-disk
 ```
+
+## Как запустить виртуальную машину при включенных политиках безопасности Admission-policy-engine
+
+Перед началом рекомендуется ознакомиться с работой модуля [Admission-policy-engine](https://deckhouse.ru/documentation/v1/modules/015-admission-policy-engine/).  
+При настройке политик рекомендуется опираться на политики безопасности, которые установленны в Вашей компании.
+
+Рассмотрим на примере включенной политики Baseline.  
+Так как Baseline не позволяет по умолчанию запускать Pod виртуальной машины из-за повышенных привилегий, необходимых для корректной работы виртуальной машины, то потребуется вручную настроить пространства имён, в которых они будут запускаться.
+
+1. Исключение пространства имён из политики Baseline.  
+```yaml
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kubernetes.io/metadata.name: <название пространства имён>
+    security.deckhouse.io/pod-policy: privileged
+  name: <название пространства имён>
+spec:
+  finalizers:
+  - kubernetes
+```
+2. Настройка новой политики безопасности.  
+Эта политика основана на Baseline и позволяет запускать виртуальные машины в заданном прострастве имён.
+```yaml
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: SecurityPolicy
+metadata:
+  name: virt-launcher-deny
```

Comment:
Why deny? Mb allow?

Reply:
Like d8-pod-security-baseline-deny-default. I suppose it depends on "enforcementAction" field value.